application/common/models/Email.php

```diff
@@ -163,7 +163,12 @@ public function getMessage()
                 'text' => $this->text_body
             ]
         );
-        $mailer->setFrom(\Yii::$app->params['fromEmail']);
+        $from = \Yii::$app->params['fromEmail'];
+        $name = \Yii::$app->params['fromName'];
+        if (! empty($name)) {
+            $from = [$from => $name];
```

Comment:
This is probably too picky. So, disregard it if you want to.
It seems funny to me that the value of `$from` changes from a string to an array.
Would it be cleaner to say `if (empty($name)) { $mailer->setFrom($from) } else { $mailer->setFrom([$from =>name]) }`?

Reply:
PHP is funny that way. I do like your suggestion, though.